packages/@ember/application/tests/application_test.js

```diff
@@ -369,11 +369,11 @@ moduleFor(
       this.runTask(() => this.createApplication());
 
       assert.equal(messages[1], 'Ember  : ' + VERSION);
-      if (jQuery) {
-        assert.equal(messages[2], 'jQuery : ' + jQuery().jquery);
+      if (jQueryDisabled) {
+        assert.equal(messages[2], 'my-lib : ' + '2.0.0a');
         assert.equal(messages[3], 'my-lib : ' + '2.0.0a');
```

Comment:
@cibernox this is related to the failing test, and it seems this line has to go into the else-block (which is when jQuery is enabled)! (as you have switched the order of the jQuery/no-jQuery blocks)

Reply:
Good catch, but it wasn't related with the failing test apparently.

Reply:
I think it did, as the previous failed test was related to the env output, and that seems to be passing now. But a deprecation test is now failing: https://travis-ci.org/emberjs/ember.js/jobs/384219437#L1562 🤔

Reply:
That was the only failure I was seeing before. That was the one I don't see why it is failing because of my changes